config/docker/chromeos-tast/Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN apt-get update && apt-get install -y \
     python3 \
     python-pkg-resources \
     vim \
-    xz-utils
+    xz-utils \
+    inetutils-ping
```

Comment:
It's a bit better to keep the packages in alphabetical order, that helps as the list grows.